src/static/helpers/slasHelper.ts

```diff
@@ -147,6 +147,70 @@ export async function authorize(
   return {url: redirectUrlString, ...getCodeAndUsidFromUrl(redirectUrlString)};
 }
 
+/**
+ * Wrapper for the /token endpoint. Depending on the parameters provided, will set up different grant types.
+ * @param slasClient a configured instance of the ShopperLogin SDK client
+ */
+export async function getAccessToken(
+  slasClient: ShopperLogin<{
+  shortCode: string;
+  organizationId: string;
+  clientId: string;
+  siteId: string;
+}>,
+parameters: {
+  redirectURI: string;
+  refreshToken?: string;
+  codeVerifier?: string;
+  code?: string;
+  usid?: string;
+},
+credentials?: {
+  client_id: string;
+  client_secret: string;
+}
+): Promise<TokenResponse> {
+
+  let tokenHeader = {}
+  if (credentials) {
+    const authorization = `Basic ${stringToBase64(
+      `${credentials.client_id}:${credentials.client_secret}`
+    )}`;
+
+    tokenHeader = {
+        Authorization: authorization
+    }
+  }
+
+  const grantType = parameters.refreshToken ? 'refresh_token'
+    : parameters.codeVerifier ? 'authorization_code_pkce'
+    : parameters.code ? 'authorization_code'
+    : 'client_credentials'
```

Comment:
The parameters we provide to `/token` affect the `grant_type` and what the endpoint ultimately does. 

I think the order of this conditional is the correct priority order for how to handle scenarios where multiple parameters corresponding to different grant_types are provided.